apertium-get.py

```diff
@@ -0,0 +1,388 @@
+#!/usr/bin/python3
+
+import argparse
+import os
+import re
+import sys
+import urllib.request
+from subprocess import (
+    run,
+    check_call,
+    check_output,
+    STDOUT,
+    DEVNULL,
+    CalledProcessError,
+)
+from enum import Enum
+
+### Globals:
+
+PMODULES = ["trunk", "staging", "nursery", "incubator"]
+LMODULES = ["languages", "incubator"]
+
+git_ssh = "git@github.com:"
+git_https = "https://github.com/"
+
+apertium_git = "apertium/apertium-%s.git"
+giella_git = "giellalt/lang-%s.git"
+giella_core_git = "giellalt/giella-core.git"
+giella_shared_git = "giellalt/giella-shared.git"
+
+
+class Status(Enum):
+    NOT_STARTED = 0
+    CLONED = 1
+    PULLED = 2
+    DONE = 3
+    SKIPPED = 4
+    FAILED = 5
+
+
+Dep_Paths = {}
+# e.g. 'apertium-eng' -> '~/apertium-eng'
+
+Dep_Status = {}
+# e.g. 'apertium-spa' -> Status.CLONED
+
+Dep_Reqs = {}
+# e.g. 'apertium-tur-uzb' -> [(1, 'apertium-tur'), (2, 'apertium-uzb')]
+
+AP_CHECK_LING = re.compile(r"AP_CHECK_LING\(\[(\d)\],\s+\[([\w-]+)\]", re.MULTILINE)
+# original pattern:
+# (awk -F'[][[:space:]]+' '/^ *AP_CHECK_LING\(/ && $2 && $4 {print $2, $4}' "${pair}"/configure.ac)
+
+
+def get_output(command, dirname=None):
```

Comment:
Rather than having a single arg working, this should take `**kwargs` and splat that into `check_output`.